market_api/market_api.py

```diff
@@ -0,0 +1,29 @@
+import requests
+
+BASE_URL = "http://api.marketcheck.com"
+API_URL = "/v1/history/2FMGK5D81EBD14330"
+
+class MarketAPI(object):
```

Comment:
Since api_key is common for all the calls,  it is better to separate  in a superclass and all the actual API calls, like `MarketAPI` just subclass from it.